exchanges/poloniex/poloniex.go

```diff
@@ -1097,9 +1100,9 @@ func (e *Exchange) SendAuthenticatedHTTPRequest(ctx context.Context, ep exchange
 				}
 			}
 			if string(bodyPayload) != "{}" {
-				signatureStrings = fmt.Sprintf("%s\n%s\n%s&%s", method, endpoint, "requestBody="+string(bodyPayload), values.Encode())
+				signatureStrings = fmt.Sprintf("%s\n%s\n%s&%s", method, path, "requestBody="+string(bodyPayload), values.Encode())
```

Comment:
🤷‍♂️ Just noting that this one is really weird, because you're concating inside the arguments to Sprintf.
And also using `string()` which is like ... what sprintf would be for, if you're going to use it.